eldritch/characters.py

```diff
@@ -219,10 +219,12 @@ def get_spend_event(self, handle):
       return events.ReturnMonsterToCup(self, handle)
     return events.ReturnGateToStack(self, handle)
 
-  def bonus(self, check_name, state, attributes=None):
+  def bonus(self, check_name, state, attributes=None, check=True):
     modifier = state.get_modifier(self, check_name)
     for pos in self.possessions:
       bonus = pos.get_bonus(check_name, attributes)
+      if check:
```

Comment:
I think you will need to also do the following:
- update the lantern to only give +1 to luck checks
- update all mythos cards that affect checks
- update the Check event to use the new _check attribute

I'm also open to other ideas on how to distinguish between checks/raw attributes.